packages/apps-config/src/endpoints/testingRelayWestend.ts

```diff
@@ -107,19 +107,6 @@ export const testParasWestend: Omit<EndpointOption, 'teleport'>[] = [
       color: '#03f3f3',
       logo: nodesKhalaSVG
     }
-  },
```

Comment:
Just needs a removal of the actual (now unused) png file as well in the import on line 8.

Reply:
Fixed in [39d90b5](https://github.com/polkadot-js/apps/pull/10032/commits/39d90b5a78fefcc69dddac3be163ec0aea2ea66e)